content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,416 @@
+
+But most importantly, an important cleanup has been done, solving some bugs. For example, updating a tag could require restarting darktable to be taken into account; this is no longer necessary. In short, darktable 3.0 initiated a new keyword module, darktable 3.2 refines and finalizes it.
+
+## New collection filters
+
+Seven new filters are added to the many already existing ones. darktable is already very rich in filters on photo metadata (EXIFS data and IPTC/XMP data). On the other hand, very few filters existed on the various events or treatments specific to darktable. You already knew the filters **film roll**, **tag** et **local copy**. darktable 3.2 release add now these new following filters: **history**, **module**, **module order**, **import timestamp**, **change timestamp**, **export timestamp** and **print timestamp**. Here is a brief description of the role of these new filters.
+
+* **history** - allow to select altered or not altered photos (this include photos with auto-applied presets).
+* **module** - allow to select photos where development stack contains a precised module. All modules used at least once are presented.
+* **module order** - allow to select photos altered with a precise module order. All existing orders are shown.
+* **import timestamp** - allow to select imported photos with a specific timestamp (timestamp = date and time). All timestamps are shown.
+* **change timestamp** - allow to select altered photos at a specific timestamp. All timestamps are shown.
+* **export timestamp** - allow to select exported photos at a specific timestamp. All timestamps are shown.
+* **print timestamp** - allow to select printed photos at a specific timestamp. All timestamps are shown.
+
+## New image informations
+
+The image information module also displays new information. The four timestamps mentioned above : **import timestamp**, **change timestamp**, **export timestamp** et **print timestamp**, as well as new EXIF information: the **exposure bias**.
+
+Module window is smaller, but module is now scrollable et resizable with `Ctrl` + `scroll`
+
+# A new module : _negadoctor_
+
+The current silver negative _invert_ module has a major flaw: it necessarily works on sensor'signal that has not been demosaicing, if the negative was scanned with a digital camera, and therefore necessarily before the input color profile, no matter what happens. This implies two insoluble problems :
+
+1. If the digital file comes from a real scanner, and it is encoded with a transfer function (the "gamma"), it is not cancelled before the inversion. Depending on the type of scanning you are working on ("RAW" 16-bit linear scan from a specialized film scanner, or 8-bit gamma encoded JPEG scan from a consumer scanner), you will not have the same behaviour at all. Note that if the scan is made from another device, the signal is linear, so the problem does not arise here.
```

Comment:
it is not cancelled before the inversion -> it is not removed before the inversion.

Note that if the scan is made from another device -> Note that if the scan is made from a digital camera